cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -443,7 +443,7 @@ install() {
         add_install_dir_to_ci_path "$_install_dir"
         add_install_dir_to_path "$_install_dir_expr" "$_env_script_path" "$_env_script_path_expr" ".profile" "sh"
         exit1=$?
-        add_install_dir_to_path "$_install_dir_expr" "$_env_script_path" "$_env_script_path_expr" ".bash_profile .bash_login .bashrc" "sh"
+        shotgun_install_dir_to_path "$_install_dir_expr" "$_env_script_path" "$_env_script_path_expr" ".bash_profile .bash_login .bashrc" "sh"
```

Comment:
I believe this *slightly* changes behaviour here. Currently we will create `.bash_profile` if none of these files exist, and with shotgun we no long detect this case. Is that intentional?

Reply:
No, it's not - good catch, thanks! I'll fix that.